cmake/modules/dts.cmake

```diff
@@ -83,6 +89,9 @@ find_package(Dtc 1.4.6)
 # - DTS_SOURCE: the devicetree source file to use may be pre-set
 #   with this variable; otherwise, it defaults to
 #   ${BOARD_DIR}/${BOARD.dts}
+# - DTS_SOURCE: the devicetree source file to use may be pre-set
+#   with this variable; otherwise, it defaults to
+#   ${BOARD_DIR}/${BOARD.dts}
```

Comment:
Duplicate comment, and above needs to be adjusted to `${BOARD}.dts`

Reply:
How did that get duplicated  😮 

And fixed old error 😉